app.js

```diff
@@ -51,16 +51,34 @@ testMultiply(5,9);
 Write a function called sumAndMultiply() that takes in three numbers as separate arguments and returns an array where the first element is the sum of those three numbers, the second element is the product of those three numbers,  and the third and fourth elements are strings that EXACTLY follow this example and use the values that were input into the function:
 Third element: "4 and 7 and 5 sum to 16."
 Fourth element: "The product of 4 and 7 and 5 is 140."
-IMPORTANT DETAIL: You may not use the arithmetic operators + and * in this function. To do addition, use your sum() function, and to do multiplication, use your multiply() function that you've already created. You're going to have to be resourceful to figure out how to do this.
-Test this function by hand in the console to get it working, and when you think it is finished, uncomment the call for the testSumAndMultiply() function and see if the test passes.*/
+IMPORTANT DETAIL: You may not use the arithmetic operatod when you think it is finished, uncomment the call for the testSumAndMultiply() function and see if the test passes.*/
 
 // Write your code here
-function sumAndMultiply(a,b,c){ //eslint-disable-line
+function sumAndMultiply(a,b,c){
+  var array = [a,b,c];
```

Comment:
Interesting... when you put these values in an array (which no one else did, by the way), I expected you were planning to use the `sumArray` and `multiplyArray` functions to calculate the values.

If you want some bonus points, resubmit this without using `sum` and `multiply` directly (and still no arithmetic operators, of course).